src/main/java/co/omise/models/Dispute.java

```diff
@@ -13,6 +15,7 @@
     private DisputeStatus status;
     private String message;
     private String charge;
+    private Map<String, Object> metadata;
```

Comment:
Not sure if we should enforce Immutability here?

Reply:
Not sure what benefit to change it?

@chakrit What do you think about this?